packages/InteractiveProfilingTool-Core.package/MessageTally.extension/instance/gcStats.st

```diff
@@ -0,0 +1,4 @@
+*InteractiveProfilingTool-Core
```

Comment:
@Alexander-Dubrawski: Du kannst bestehende Klassen erweitern, sodass die Änderung trotzdem in deinem Package getrackt wird, indem du die ergänzten Methoden in eine Kategorie einordnest, die exakt der Name des Packages mit einem `*` gepräfixt ist.

Das sollte natürlich trotzdem nur in Ausnahmefällen gemacht werden, immerhin besteht die Gefahr von Konflikten mit anderen Packages.